runtime/manager_test.go

```diff
@@ -4209,7 +4210,8 @@ func TestNewManagerFromReaders(t *testing.T) {
 					},
 					P2P: &configs.P2PConfig{
 						PrivateKey:      "0ca1a40ddecdab4f5b04fa0bfed1d235beaa2b8082e7554425607516f0862075dfe357de55649e6d2ce889acf15eb77e94ab3c5756fe46d3c7538d37f27f115e",
-						ConsensusPort:   8080,
+						Hostname:        "node1.consensus",
+						Port:            defaults.DefaultP2PPort,
```

Comment:
Why do we use `defaults.DefaultP2PPort` in one place and hardcode `42069` everywhere else?

Reply:
Good question. Except for two instances, all other usage is inside of string literals. It didn't seem prudent to rewrite the tests to interpolate `defaults.DefaultP2PPort` into what's historically been (many) simple string literals. Do you disagree?

Of those two exceptions, one has a corresopnding string literal representation of itself in the test case which it asserts against; see the [`unset MaxMempoolCount should fallback to default value` sub-test](https://github.com/pokt-network/pocket/blob/main/runtime/manager_test.go#L4242). I would argue that this  one should be left alone due the expected literal correspondence.

I've pushed a commit for the other instance: 743d5cb0.

Reply:
Makes sense to me. Ty for the explanation